test/zmumu/tp_from_aod_Data.py

```diff
@@ -192,15 +188,15 @@
         AllVariables,
         ExtraIsolationVariables,
         PuppiIsolationVariables,
-        MVAIsoVariablesPlain,
         isoTrk03Abs = cms.InputTag("probeMuonsIsoValueMaps","probeMuonsIsoFromDepsTk"),
         isoTrk03Rel = cms.InputTag("probeMuonsIsoValueMaps","probeMuonsRelIsoFromDepsTk"),
         dxyBS = cms.InputTag("muonDxyPVdzmin","dxyBS"),
         dxyPVdzmin = cms.InputTag("muonDxyPVdzmin","dxyPVdzmin"),
         dzPV = cms.InputTag("muonDxyPVdzmin","dzPV"),
-        PtRatio= cms.InputTag("AddPtRatioPtRel","PtRatio"),
-        PtRel= cms.InputTag("AddPtRatioPtRel","PtRel"),
-        radialIso = cms.InputTag("radialIso"), 
+        JetPtRatio= cms.InputTag("AddLeptonJetRelatedVariables","JetPtRatio"),
```

Comment:
What Giovanni and Hugues say is that you are not putting anything from "AddLeptonJetRelatedVariables" into test/zmumu/tp_from_aod_MC.py